packages/jaeger-ui/src/components/TracePage/TracePageHeader.css

```diff
@@ -19,11 +19,26 @@ limitations under the License.
   display: flex;
 }
 
+Divider {
```

Comment:
I don't think this targets anything...?

Reply:
Sorry I was testing the antd divider with the style because I can't see it. I'll remove it